package-parser/tests/commands/generate_annotations/unused_functions_list.json

```diff
@@ -0,0 +1,6 @@
+[
+    "sklearn.base._BaseEstimator.__setstate__",
+    "sklearn.base.is_regressor",
+    "sklearn.cluster._agglomerative.linkage_tree",
+    "sklearn.cluster._kmeans.MiniBatchKMeans.init_size_"
+]
```

Comment:
Gehoert natuerlich eigentlich nicht ins Repo. Kann aber erstmal hier bleiben und dann spaeter beim integrieren von der Funktion in den Programmablauf entfernt werden, da sie dann nicht mehr notwendig sein wird.